### ISSUE TITLE
Zibo mod - Pushback failure, aircraft appears to not have any steerable gears

### ISSUE THREAD

**Issue by WebMaximus** [skiselkov/BetterPushbackC#230]
Noticed whenever I load a saved situation with the Zibo mod I get a repeated aural error message saying

'Pushback failure. The aircraft appears to not have any steerable gears'

This only happens when loading a saved situation and BetterPushback still seems to work without any issues. A bit annoying though to get this error message over and over again whenever loading a saved situation.

**Comment by RangerGH** [comment]
I'm seeing the same thing with version 0.47 with both default aircraft (C172) and FlyJSim B727.  I get the message when I start the sim and the default situation is loading .  Everything seems to work but  it's annoying as WebMaximus said and I wonder if there is an issue.

**Comment by WebMaximus** [comment]
Actually, I forgot about this issue since I haven't seen (heard) it for a very long time.

I'm currently on X-Plane 11.41, BetterPushback 0.47 and Zibo 3.39.31

**Comment by RangerGH** [comment]
That is understandable.  I did a search for the issue and ended up here so I added to this thread.  I'm on 11.41r1 (build 114100 64-bit).  If you need any information or testing let me know.   Other than that it appears to work well though I've only used it a few times but plan on using it a lot more.

**Comment by RangerGH** [comment]
It doesn't happen on the default C172 and only happens when starting X-Plane and loading a situation.  I've been using the FJS 727 as my default.

**Comment by slicker55** [comment]
It's been a few months since the last update to this thread so I have contacted @skiselkov to see if he knows why this happens and how to fix it

**Comment by RangerGH** [comment]
Thanks.  I've been meaning to get back to this but haven't.

**Comment by slicker55** [comment]
Thank you for an almost instant response. I hope you will be able to look at this issue soon. I am happy to assist if you need anyone to test anything out.

**Comment by WebMaximus** [comment]
I'm the OP and strange how I haven't had this issue for a very long time now.

I'm still in the Zibo mod and I'm still always loading a saved situation. Would be interesting to know what is causing the problem for those of you who still face this issue.

**Comment by slicker55** [comment]
I believe you're saying you 'used' to have the problem some time ago and somehow it has righted itself?
Interesting, .. there's hope for the rest of us!
I wonder if you're doing something different to what you used to do?

**Comment by WebMaximus** [comment]
That's correct, as said I haven't seen (or rather heard) this issue for ages.

Sorry to say I don't recall if I managed to fix it somehow or if it disappeared by itself. I do everything as I always did.

Only suggestion I have is to of course make sure you're on the latest version of Better Pushback. But I'm sure you already are.

I'll keep my fingers crossed you find a cure!

**Comment by slicker55** [comment]
thanks for the moral support... appreciated... yes, using latest of everything... just thought I was on to something... tried loading with engines running but no, false optimism... search continues...
Out of interest, maybe you can help to nail this ... can you tell me what airport you're starting out at cold and dark... and, I assume you're loading the saved situation via the x-plane main menu?

**Comment by WebMaximus** [comment]
Works fine regardless of airport and yes, loading via the main menu

**Comment by slicker55** [comment]
Roger :)

**Comment by RangerGH** [comment]
I get it starting up X-Plane no matter what plane I have.  Let me try some more testing.

**Comment by slicker55** [comment]
Thanks... really appreciate your fast response...

**Comment by RangerGH** [comment]
As I said.  I'm not the author so I can't fix it but I can give him more data.

**Comment by slicker55** [comment]
Yes, I understand that - just wanted to thank you for doing what you can to help - I suspect the more info we can pass on, the better

**Comment by slicker55** [comment]
Hello,

Just wondering you have had time to look at this issue yet?

I am happy to assist if you need anyone to test anything out.

John

**Comment by RangerGH** [comment]
Sorry, life got busy.  I did test it but didn't get around to posting it.  I'm seeing the message about missing steerable nosewheel only when I start X-Plane and the aircraft loading is about done.  If I switch aircraft I do not get it.

Hopefully the author can look at it.

**Comment by slicker55** [comment]
just updating this thread having installed the latest zibo patch (3.43.4) - this issue persists but format has slightly changed - 30 seconds before Xplane has fully loaded up, an audible message from a lady says,  'pushback failure', she only used to say this twice but now repeats 6 or 7 times before saying, 'no steerable gear'... zibo is fine once loaded... just annoying not knowing how to stop this happening... strange, no-one has figured it out yet - fingers crossed, I'm sure someone will :)

**Comment by RangerGH** [comment]
I still get this, too.  Not sure how many times but it is only on startup of X-Plane and any plane. Doesn't affect the function but it is annoying.

**Comment by slicker55** [comment]
ok... so I discovered something new today :)
the ladies warning message... 'appears not to have any steerable gear'... is triggered if you add A.I planes via the 'custom tab' ... there was a clue when she suddenly started repeating the same warning message ... I added nine A.I's last to check out something I saw on a Youtube tutorial ... if you look at the text on the bottom left-hand side of the screen as X-Plane loads, you see each rendition of the message coincides with each plane as it loads up... 
the same happens if you choose A.I planes from the Laminar Research ... so, at least the mystery is part-solved :)

**Comment by RangerGH** [comment]
I haven't added any AI planes.

**Comment by slicker55** [comment]
suggest you check to see if there are any in there by default ... mine did have some LR ones which I deleted before adding A.I ones I got from a pack downloaded from xplane.org

**Comment by RangerGH** [comment]
You are correct.  It had three.  I'm replacing BetterPushback anyway so it is a moot point.

**Comment by slicker55** [comment]
out of interest, what are you replacing BPB with?

**Comment by RangerGH** [comment]
I am using FS-FlightControl that a friend told me about.  It is payware but works for all the simulators - MSFS, P3D, FSX, X-Plane and does more than just pushback.  For me it's worth it as I fly  P3D for one or two specific aircraft so even though BP is free I'd have to buy something for P3D and maybe eventually MSFS so FS-FlightControl gives me pushback for all my  simulators and some more stuff which I will use some of it at least.  

Plus, we haven't heard any response from the author on BP so I'm not sure what the status of it is.

**Comment by slicker55** [comment]
fair comment.. always on the lookout for new stuff... I have heard of FS-FlightControl , in fact, a guy used it on one of the YT vids a few days ago... will take a closer look but for me, BP works (most of the time) LOL....

**Comment by RangerGH** [comment]
BP works once it starts up and the voices are just annoying but it would be nice to get a response.  

Another plus for FS-FlightControl is that it handles taildraggers and I fly a DC-3 a lot.  BP can't handle the DC-3.

**Comment by slicker55** [comment]
agreed...

**Comment by slicker55** [comment]
Just came across this old post and wondered if there was any update/fix etc... 
To recap, you will hear a warning _“Pushback failure, aircraft appears to not have any steerable gears”,_ if you have added A.I planes via the 'custom tab'...
